### ISSUE TITLE
Regression Performance Tab has ZeroDvisionError when target values have zero values

### ISSUE THREAD

**Issue by yanhong-zhao-ef** [evidentlyai/evidently#18]
I have a dataset with some zeros values in target and this line is throwing error the Zero division zero
```
abs_perc_err = list(map(lambda x : 100*abs(x[0] - x[1])/x[0], zip(reference_data[target_column], reference_data[prediction_column])))
ZeroDivisionError: float division by zero
```
I was thinking since the datasets are already in DataFrame can't we do something like this
```
abs_perc_err = ((reference_data[target_column]-reference_data[prediction_column])/reference_data[target_column]*100).abs()
```
and the zero target values will now be returned with -inf instead

**Comment by emeli-dral** [comment]
Hi @yanhong-zhao-ef, thank you for pointing this out, this is a bug!
This is an old piece of code, it will be replaced by vectorised version in the next package.

**Comment by emeli-dral** [comment]
fixed in ```evidently==0.1.15.dev0```
